actions/app.go

```diff
@@ -88,25 +79,22 @@ func App() *buffalo.App {
 		}
 		app.Use(T.Middleware())
 
-		// serve go-get requests
-		app.Use(GoGet(cdnGetter))
 		app.GET("/", homeHandler)
 
-		app.GET("/{base_url:.+}/{module}/@v/list", listHandler(storage))
-		app.GET("/{base_url:.+}/{module}/@v/{version}.info", versionInfoHandler(storage))
-		app.GET("/{base_url:.+}/{module}/@v/{version}.mod", versionModuleHandler(storage))
-		app.GET("/{base_url:.+}/{module}/@v/{version}.zip", versionZipHandler(storage))
-		app.POST("/admin/upload/{base_url:[a-zA-Z./]+}/{module}/{version}", uploadHandler(storage))
-		app.POST("/admin/fetch/{base_url:[a-zA-Z./]+}/{owner}/{repo}/{ref}/{version}", fetchHandler(storage))
-
-		auth := app.Group("/auth")
-		auth.GET("/{provider}", buffalo.WrapHandlerFunc(gothic.BeginAuthHandler))
-		auth.GET("/{provider}/callback", authCallback(mgoStore))
-		//	app.GET("/{base_url:.+}/{module}", homeHandler)
-		// serve files from the public directory:
+		if MODE == "proxy" {

```

Comment:
we have default set to proxy proxy should be also when invalid option is passed.
i would do explicit check for registry if(registry) else ( is proxy does not matter on value)

Reply:
done - I added a check for unsupported mode